charts/kubewarden-controller/templates/deployment.yaml

```diff
@@ -19,6 +21,7 @@ spec:
         {{- if .Values.telemetry.enabled }}
         "sidecar.opentelemetry.io/inject": "true"
         {{- end }}
+        {{- include "kubewarden-controller.annotations" . | nindent 8 }}
```

Comment:
```suggestion
```
Removing this `Deployment spec.template.metadata.annotations`, that will get included in the Pods. Afterall, the pods are managed by the deployments and so.